cantabular-import/deps.yml

```diff
@@ -88,3 +88,14 @@ services:
     environment:
       - VAULT_DEV_LISTEN_ADDRESS=0.0.0.0:8200
       - VAULT_DEV_ROOT_TOKEN_ID=0000-0000-0000-0000
+  kafdrop:
+    image: 'obsidiandynamics/kafdrop'
+    restart: "no"
+    ports:
+      - "9090:9000"
+    environment:
+      KAFKA_BROKERCONNECT: "kafka-1:19092"
```

Comment:
Adding the below line in environment after KAFKA_BROJERCONNECT fixed the issue for kafdrop to work
`JVM_OPTS: "-Xms16M -Xmx48M -Xss180K -XX:-TieredCompilation -XX:+UseStringDeduplication -noverify"`

Reply:
This has been addressed.